pkg/storage/constants/constants.go

```diff
@@ -26,4 +26,5 @@ const (
 	DefaultGCInterval       = 1 * time.Hour
 	S3StorageDriverName     = "s3"
 	LocalStorageDriverName  = "local"
+	GlobalBlobsRepo         = "_blobstore"
```

Comment:
Make sure this blobs repo is per substore. Because substores may be on different partitions.